9Jun2016

mikeSheets opened the pull request:
Cart and product page angularjs is working!

mikeSheets committed:
Thursday push

Update

mikeSheets committed:
Remove ignored files

mikeSheets committed:
Remove ignored files

mikeSheets committed:
remove tmp and log

mikeSheets committed:
remove idea

mikeSheets committed:
Tuesday Afternoon commit

just a quick push

mikeSheets committed:
Remove ignored files

mikeSheets committed:
Wednesday Commit

Just updating where I’m at so far.

mikeSheets committed:
Safety commit

Pushing to github before I link github to intellij

mikeSheets committed:
Test commit

mikeSheets committed:
Test commit #2

mikeSheets committed:
api/v1/ controllers updated

mikeSheets committed:
Friday 24JUN2016

Cart and product page angularjs is working!

mikeSheets committed:
api/v1/ controllers updated

fiestacasey commented on app/assets/javascripts/angularjs.coffee:
```diff
@@ -0,0 +1,118 @@
+
+# order resource
+
+app.factory('Order', ['$resource', ($resource) ->
```
you should start splitting this stuff out into multiple files, they're getting pretty big.

fiestacasey commented on app/assets/javascripts/products.coffee:
```diff
@@ -1,3 +1,17 @@
+# All this logic will automatically be available in application.coffee.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).on 'ready', ->
```
you shouldn't need this once your ng stuff is good

fiestacasey commented on app/assets/stylesheets/cart.scss:
```diff
@@ -1,3 +1,89 @@
+        height: 30px;
+        margin: 0 auto;
+      }
+      .update{
```
empty selector

fiestacasey commented on app/controllers/admin/products_controller.rb:
```diff
@@ -17,15 +17,19 @@ def show
     # GET /products/new
     def new
       @product = Product.new
```
shouldn't need this line. it is created by cancancan with the load_resource statement above

fiestacasey commented on app/controllers/api/v1/orders_controller.rb:
```diff
@@ -0,0 +1,6 @@
+class Api::V1::OrdersController < ApplicationController
+
+  def get_cart
+    render json: cart.to_json
```
now that the CartHelper#cart function is called on every page we should change it so that it doesn't create the cart unless we need it to. AKA; if someone hits the homepage but never tries to add anything to their cart it shouldn't save a cart.

mikeSheets committed:
Checkout functionality nearing complete. Still some TODOs

mikeSheets committed:
Working on final check out

Working on splitting the angular into multiple files and the payment
and inventory stuff.

mikeSheets committed:
Added user login requirement to checkout, got the angularJS files importing properly, started the angularjs for the placing order business

mikeSheets committed:
Fixed some order item handling stuff, got the inventory check and adjustment working, changed the empty cart button path back to products page.  Payments appears to be working, but still checking function and debugging.
